taxcalc/functions.py

```diff
@@ -405,10 +412,9 @@ def AMED(_fica, e00200, MARS, AMED_thd, _sey, AMED_trt,
     """
 
     # ratio of income subject to AMED tax = (1 - 0.5*(FICA_mc_trt+FICA_ss_trt)
-    _amed = AMED_trt * max(0, e00200 +
-                           max(0, _sey) * (1 - 0.5 *
-                                           (FICA_mc_trt + FICA_ss_trt)) -
-                           AMED_thd[MARS - 1])
+    _amed = AMED_trt * (max(0, e00200 - AMED_thd[MARS - 1]) + max(0, max(0,
```

Comment:
@feenberg 
The `0.5` doesn't come along, and is always associated within the expression `(1 - 0.5*(FICA_mc_trt+FICA_ss_trt))`, which yields a value of `0.9235`. And the while expression `max(0, sey) *(1 - 0.5*(FICA_mc_trt+FICA_ss_trt))`, which is equivalent to the variable `_sey9235 ` in SAS calc, corresponds to line 8 in form 8959. Note this is a copy of line 4 (or line 6) in form Sch SE, and is subject to a rate of 92.35%. Hope this addressed your concern.